### PR TITLE
Bump baseline version to 0.14.0

### PR DESCRIPTION
**Pull request by gatesn**
checkstyle -> errorprone

### CODE REVIEW

**Review by lwwmanning — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -12,11 +12,11 @@
 
 buildscript {
     repositories {
-        jcenter()
+        maven { url  "http://palantir.bintray.com/releases" }
```
Comment (mnazbro) [inline build.gradle:15]: You shouldn't need to add bintray here. Check out https://github.com/palantir/auth-tokens/pull/33/files#diff-c197962302397baf3a4cc36463dce5eaR14

You may need to add the gradle plugin repo

File: src/main/java/com/palantir/python/miniconda/AfterEvaluateAction.java [hunk 2]
```diff
@@ -64,7 +64,7 @@ public AfterEvaluateAction(
     }
 
     @Override
-    public void execute(Project project) {
+    public final void execute(Project project) {
```
Comment (mnazbro) [inline src/main/java/com/palantir/python/miniconda/AfterEvaluateAction.java:67]: It's probably easier for the future to make the class final.

File: src/main/java/com/palantir/python/miniconda/MinicondaExtension.java [hunk 3]
```diff
@@ -57,7 +57,7 @@ public MinicondaExtension(Project project) {
         this.project = project;
     }
 
-    public void validate() {
+    public final void validate() {
```
Comment (mnazbro) [inline src/main/java/com/palantir/python/miniconda/MinicondaExtension.java:60]: Same comment as the other one

File: src/main/java/com/palantir/python/miniconda/AfterEvaluateAction.java (hunk 2 above)
Comment (gatesn) [reply src/main/java/com/palantir/python/miniconda/AfterEvaluateAction.java:67]: Gradle creates proxies for the classes so can't make it final :(

File: build.gradle (hunk 1 above)
Comment (gatesn) [reply build.gradle:15]:
```> Could not resolve all dependencies for configuration ':classpath'.
   > Could not find gradle.plugin.com.palantir.baseline:gradle-baseline-java:0.14.0.
     Searched in the following locations:
         https://plugins.gradle.org/m2/gradle/plugin/com/palantir/baseline/gradle-baseline-java/0.14.0/gradle-baseline-java-0.14.0.pom
```

**Review by mnazbro — APPROVED** [review]

**Pull request merged**